fastmri/data/mri_data.py

```diff
@@ -206,6 +208,8 @@ def __init__(
         volume_sample_rate: Optional[float] = None,
         dataset_cache_file: Union[str, Path, os.PathLike] = "dataset_cache.pkl",
         num_cols: Optional[Tuple[int]] = None,
+        annotations: bool = True,
```

Comment:
Many researchers are used to the current functionality without annotations. This should be false by default.

Also, `use_annotations` is a better name as it more clearly defines the purpose.

```suggestion
        use_annotations: bool = False,
```

Reply:
In this case, if user is using the subclass AnnotatedSliceDataset that inherits the result of the SliceDataset, is it not necessary to input use_annotation?